tests/flow/test_temporal.py

```
@@ -0,0 +1,550 @@
+        #-----------------------------------------------------------------------
+        # duration + duration
+        #-----------------------------------------------------------------------
+
+        q = """RETURN duration({years:1, months:1, weeks:1, days:1, hours:1, minutes:32, seconds:10}) +
+                      duration({years:2, months:2, weeks:2, days:2, hours:2, minutes:34, seconds:12})"""
+        res = self.graph.query(q).result_set[0][0]
+        self.env.assertEquals(res, timedelta(years=3, months=3, days=24, hours=4, minutes=6, seconds=24))
+
+        return
+
+        #-----------------------------------------------------------------------
```

Flake8 (7.2.0)

[error] block comment should start with '# '

(E265)